tests/unit/config/test_config.py

```diff
@@ -19,6 +19,7 @@
     is_section_substitution,
     parseconfig,
 )
+from tox.config.parallel import ENV_VAR_KEY as PARALLEL_ENV_VAR_KEY
```

Comment:
I can see this was marked as resolved, but this is the way it's being used everywhere at the moment. I think the reason is that `ENV_VAR_KEY` is not a clear constant name representing parallel mode

Reply:
Yeah, this was is clearer.

Reply:
So we should rename ENV_VAR_KEY to PARALLEL_ENV_VAR_KEY globally or is there some backwards compatibility problem with this already?

Reply:
I think it's fine as it is. Within the module, ENV_VAR_KEY is descriptive enough (namespaced). But when you import it from otherwise you can use aliasing to make it clearer its origin.